packages/marketplace/src/components/ui/developer-analytics/billing/cost-explorer/cost-filter-form.tsx

```diff
@@ -15,6 +17,11 @@ export const handleAutoSave = (onSave: (values: CostFilterFormValues) => void) =
 }
 
 const CostFilterForm: React.FC<CostFilterFormProps> = ({ initialValues, onSave }) => {
+  const myIdentity = useSelector(selectMyIdentity)
+
+  const minDate = myIdentity.created && new Date(myIdentity?.created)
+  const maxDate = new Date()
```

Comment:
suggest to use dayjs instead of new Date()

Reply:
As I saw as https://github.com/Hacker0x01/react-datepicker/blob/master/docs/datepicker.md.
react-datepicker only accept minDate, maxDate of Date obj.